src/App.js

```diff
@@ -1,14 +1,88 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-import { Loop, Stage } from 'react-game-kit';
+import { Loop, Stage, Body, World } from 'react-game-kit';
+import Player from './player';
+import playerSprite from './assets/character-sprite.png';
 
 class App extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
If we have the `class-properties` transform, we should be able to put the default state on the class itself:

```js
class App extends Component {
  state = { /* default state */ }
}
```

Reply:
Done!